File: backend/Entities/Room.cs
namespace Cinema.Entities;

public class Room
{
    public string RoomId { get; set; }

    public string Name { get; set; }

    public string Address { get; set; } = string.Empty;

    public virtual ICollection<Seat> Seats { get; set; } = new List<Seat>();
}

File: frontend/src/types/Room.ts
import Seat from './Seat';

export default interface Room {
    roomId: string,
    name: string,       // Tên phòng, ví dụ: Phòng chiếu số 1
    address: string,    // Địa chỉ, ví dụ: Tầng 1, phòng 101
    seats: Array<Seat>  // Danh sách ghế trong phòng
}
